packages/std-client/src/utils.ts

```diff
@@ -117,16 +99,60 @@ export function resolveRelationshipChain(
   return entity;
 }
 
+export function findEntityWithComponentInRelationshipChain(
```

Comment:
can we add a short code documentation comment on top of this and `findInRelationshipChain` to explain what it does and how to use it? (then it will show up here: https://mud.dev/packages/std-client/api and be discoverable via the documentation search bar)

Reply:
for sure!